### DISCUSSION TITLE
Support for Markdown Rendering using something like Glow

### DISCUSSION THREAD

**Discussion by PierreGeldenhuys** [jucasoliveira/terminalGPT#63] [Ideas]
Hi Lucas

I'm a dev and really appreciate this project!

Is it possible to support markdown when printing out answers?
This does a pretty awesome job: https://github.com/charmbracelet/glow.

**Comment by PierreGeldenhuys** [comment]
![CodeSnippetWithGlow](https://user-images.githubusercontent.com/127150950/224566175-7d825784-e5d0-4d5d-a142-a2a4a551e93f.png)

**Comment by jucasoliveira** [comment]
Markdown support added from 1.6.2
